manifests/repo.pp

```diff
@@ -41,6 +41,14 @@
         provider    => 'dnfmodule',
       }
     }
+
+    if $facts['os']['release']['major'] == '8' and ($repo == 'nightly' or versioncmp($repo, '3.2') >= 0) {
```

Comment:
With this, is it still needed to enable Ruby? Or should this be first and then the `>= 2.5` as an `elsif`.

Reply:
yea, the >=2.5 can be an eslif